manifests/n/Nushell/Nushell/0.66.0/Nushell.Nushell.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Nushell.Nushell
+PackageVersion: 0.66.0
+PackageLocale: en-US
+Publisher: The Nu Project Contributors

```

```suggestion
Publisher: The Nushell Project Developers
```
Must match control panel